mscore/scoreaccessibility.cpp

```diff
@@ -289,10 +289,10 @@ void ScoreAccessibility::currentInfoChanged()
                         staffName = e->staff()->partName();
                   if (staffName.isEmpty()) {
                         staffName = tr("Unnamed");    // for screenreader only
-                        rez = QString("%1; %2").arg(rez).arg(staff);
+                        rez = QString("%1; %2").arg(rez.replace('\n', ' ')).arg(staff);
```

Comment:
Can't we use `↵` (0x021b5) instead if space?

Reply:
I suppose, but doesn't Mu4 use spaces also?

Reply:
It probably does, but does that make it better?
It is used for accessibility,so for screen readers, using that return arrow might work better than a space there.
Maybe we should try to get that accross upstream ;-)

Reply:
Hell, you might be onto something here :)

Reply:
I don't see any "replace \n with space" upstream, no idea how they handle this

Reply:
Ok. Certainly something happens though (no new lines with fingering text)

Reply:
Yes, I see that. Just not why and where

Reply:
One problem is that it looks like a regular staff text already "removes" newlines by its own code outside of score accessibility, so there'll be a slight inconsistency by switching newlines to downright arrows here... like staff text will still have replaced them with spaces...

Reply:
uses Qt's QString::simplified() (https://doc.qt.io/qt-6/qstring.html#simplified) Wouldn't be surprised if that was Mu4 does

Reply:
> One problem is that it looks like a regular staff text already "removes" newlines by its own code outside of score accessibility, so there'll be a slight inconsistency by switching newlines to downright arrows here... like staff text will still have replaced them with spaces...

Yes, just found that out too, in title or composer text, etc,

Reply:
> uses Qt's QString::simplified() (https://doc.qt.io/qt-6/qstring.html#simplified) Wouldn't be surprised if that is what Mu4 does somewhere

And yes, that's the most likely explanation.

Reply:
and so we should do it here too

Reply:
So let's make it 
```cpp
            statusBarLabel->setText(rez.simplified()); // no newlines etc. in the status bar
```

Reply:
So no go with ↵?

Reply:
Consistency trumps it